Initial version of Python SDK for Service Connectors

sfitts opened the pull request:
This is the initial cut at the SDK used to create Python serivce connectors.  There is also some basic initial infrastructure for the project, though I expect that to change based on Java work that is going on concurrently.

No issue logged...

sfitts committed:
Initial version of Python SDK for Service Connectors

This is the initial cut at the SDK used to create Python serivce
connectors.  There is also some basic initial infrastructure for the
project, though I expect that to change based on Java work that is going
on concurrently.

No issue logged...

mer2github commented on pythonsdk/README.md:
```diff
@@ -0,0 +1,14 @@
+## Repository Contents
+
+This repository contains an SDK for building Vantiq Service Connectors in Python. Service connectors allow some of all
```
```suggestion
This repository contains an SDK for building Vantiq Service Connectors in Python. Service connectors allow some or all
```

mer2github reviewed (approved):
LGTM - kicking myself a little for not pushing the build.gradle from the mongodb atlas effort, but it looks like your build file is eerily similar. merging will be straightforward.

sfitts commented:
> LGTM - kicking myself a little for not pushing the build.gradle from the mongodb atlas effort, but it looks like your build file is eerily similar. merging will be straightforward.

Happy to let you go first and handle the merge on my end.  I need to get deployment working before I can merge in any case.

sfitts committed:
Move requirements to requirements.txt

This will make it easier to integrate with the IDE and other tools (and
the plug-in supports this without need for duplication).

Also added license file and some other cleanup

sfitts committed:
Some tweaks

sfitts committed:
Update FastAPI

sfitts committed:
Update pytest and reset version
